content/pages/constitution.md

```diff
@@ -172,6 +178,39 @@ All committee memebers are free to delegate their responsibilities to others as
 as they see fit, but are ultimately responsible for the completion of their
 assigned duties.
 
+## ii. Volunteers ##
+
+1.  All individuals are eligible to become volunteers.
```

Comment:
We probably should discuss it more, but yeah.
++ to men from minorities 
We've had male mentors before, and stuff.
I think maybe mentioned priority will go to giving women volunteer roles